lib/msal-common/src/request/RequestValidator.ts

```diff
@@ -93,7 +93,7 @@ export class RequestValidator {
         }
 
         // Remove any query parameters already included in SSO params
-        queryParams.forEach((value, key) => {
+        queryParams.forEach((_value, key) => {
```

Comment:
is it because it's not used?

Reply:
Exactly. And I wasn't able to build msal-common to run the unit tests because this was throwing an eslint-unused-var error.